.drone.star

```diff
@@ -0,0 +1,1455 @@
+config = {
+	'app': 'updater',
+	'rocketchat': {
+		'channel': 'builds',
+		'from_secret': 'private_rocketchat'
+	},
+
+	'branches': [
+		'master'
+	],
+	'yarnlint': True,
+
+	'codestyle': True,
+
+	'phpunit': {
+		'allDatabases' : {
+			'phpVersions': [
+				'7.2',
+			]
```

Comment:
@jasson99 you can set `'coverage': False` here.

For this first attempt at getting drone CI running, we don't need to bother about coverage. We can do that after this PR is working and merged.

Reply:
okay. Thankyou